### ISSUE TITLE
Unable to locate element Error

### ISSUE THREAD

**Issue by kevXZY** [InstaPy/InstaPy#1621]
Still getting this error while using latest version...

~/InstaPy$ sudo python3.4 /bin/start.py 
INFO [2018-03-14 09:39:21] [xxxxx]  Session started - 2018-03-14 09:39:21
Issue with cookie for user xxxxxx. Creating new cookie...
Traceback (most recent call last):
  File "/bin/start.py", line 11, in <module>
    session.login()
  File "/home/xxx/InstaPy/instapy/instapy.py", line 270, in login
    self.bypass_suspicious_attempt):
  File "/home/xxx/InstaPy/instapy/login_util.py", line 137, in login_user
    "//article/div/div/p/a[text()='Log in']")
  File "/usr/local/lib/python3.4/dist-packages/selenium/webdriver/remote/webdriver.py", line 293, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/usr/local/lib/python3.4/dist-packages/selenium/webdriver/remote/webdriver.py", line 752, in find_element
    'value': value})['value']
  File "/usr/local/lib/python3.4/dist-packages/selenium/webdriver/remote/webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python3.4/dist-packages/selenium/webdriver/remote/errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//article/div/div/p/a[text()='Log in']"}
  (Session info: chrome=65.0.3325.162)
  (Driver info: chromedriver=2.36.540471 (9c759b81a907e70363c6312294d30b6ccccc2752),platform=Linux 4.4.0-31-generic x86_64)

Please advise

**Comment by mpaolor** [comment]
same error here

**Comment by gustafsom** [comment]
this is a general problem.

Take a look at this solution to see if it will solve for you.

[](https://github.com/timgrossmann/InstaPy/pull/1610)

It still did not work for me

**Comment by gustafsom** [comment]
this is a general problem.

Take a look at this solution to see if it will solve for you.

[https://github.com/timgrossmann/InstaPy/pull/1610](url)

It still did not work for me

**Comment by gustafsom** [comment]
Now work for me

**Comment by kevXZY** [comment]
@gustafsom I don't have unfollow in my script so this solution won't work for me

**Comment by patrickdundas** [comment]
I am also having this error

```
INFO - Session started - 2018-03-22 03:01:30
Cookie file not found, creating cookie...
Error!!!!! Message: no such element: Unable to locate element: {"method":"xpath","selector":"//article/div/div/p/a[text()='Log in']"}
  (Session info: headless chrome=65.0.3325.181)
  (Driver info: chromedriver=2.37.544315 (730aa6a5fdba159ac9f4c1e8cbc59bf1b5ce12b7),platform=Linux 4.4.0-112-generic x86_64)

INFO - Session ended - 2018-03-22 03:01:32

```

**Comment by Tjorriemorrie** [comment]
The login works fine - thre must be another issue. Once the above commit is merged it should save the page so that it is possible to inspect what is going on.

**Comment by patrickdundas** [comment]
I found out what the issue could be. Every few times that you navigate to instagram.com and click "log in" it will for some reason bring you to the sign up page rather than the log in. I was able to fix this by making login_util navigate to instagram.com/accounts/login and then sleep for a few seconds in case the login form is still loading. My implementation of this is terrible though and It needs work, so I'm sure one of the regulars would probably be able to make a better version that does this.
